.github/CODEOWNERS

```diff
@@ -63,6 +63,9 @@ AbapEnvironment*                                @SAP/jenkins-library-abap
 ascAppUpload*                                   @Mau04 @inf2381
 AscAppUpload*                                   @Mau04 @inf2381
 
+credentialdiggerScan*                           @SAP/credential-digger-members @SAP/credential-digger-admins 
```

Comment:
@CCFenner @OliverNocon ,probably Marco and Slim needs write permissions? https://github.com/orgs/SAP/teams/credential-digger-admins

Reply:
Hi, I see the team has been created (not by me, so thanks to who did it :D )
I think we can leave `@SAP/credential-digger-admins` without `members` (unless you prefer to have both, but I don't get the difference in that case). And anyway it's always Slim and me